packages/input-option/src/InputOption/InputOption.stories.tsx

```diff
@@ -95,4 +92,15 @@ LiveExample.parameters = {
   chromatic: { disableSnapshot: true },
 };
 
-export const Generated = () => {};
+export const Generated = {
+  render: () => <></>,
+  parameters: {
+    generate: {
+      combineArgs: {
+        highlighted: [false, true],
+        checked: [false, true],
+        disabled: [false, true],
```

Comment:
I may be overthinking this, but can an option be highlighted/checked and disabled at the same time?

Reply:
`checked`+`disabled`, yes

`highlighted`+`disabled`, not sure, but possibly?